includes/MovePage.php

```diff
@@ -31,6 +31,8 @@
  */
 class MovePage {
 
+	const SORTKEY_LENGTH = 230;
```

Comment:
A little comment explaining why we're limiting the length of the sort key and why we decided to go with 230 characters would be nice. Especially if we're going to push this change upstream

Reply:
added